### PR TITLE
Ashwinshetttty ziyob correct

### PR DESCRIPTION
**Pull request by ashwinshetttty**
I agree to not ping anybody on Discord/Twitter/email about this pull request. Instead I will inquire by posting a new comment in the pull request if needed.
PRs are reviewed in bulk and and can take up to two weeks to be merged.

This repository is managed using an auto merge action. Please ensure your PR has no deleted lines, and it will be merged.

Please provide the following information for your token.
Please include change to the src/tokens/solana.tokenlist.json file in the PR.
DON'T modify any other token on the list.

At minimum each entry should have

Token Address:DKCc1ja8WfxL1oki7zTBKoac74GHZXLfrcazBWbDJpwD
Token Name:Ziyob
Token Symbol:ZYB
Logo:https://github.com/ashwinshetttty/token-list/blob/main/assets/mainnet/DKCc1ja8WfxL1oki7zTBKoac74GHZXLfrcazBWbDJpwD/ZiyobLogo.png
Link to the official homepage of token:https://www.ziyob.com/
Coingecko ID if available (https://www.coingecko.com/api/documentations/v3#/coins/get_coins__id_):

### CODE REVIEW

**Review by ashwinshetttty — COMMENTED** [review]
Applied a few changes

**Comment by Solana-Developer** [comment]
Please read the [SPL-token README](https://github.com/SolDev-stack/token-list/blob/main/README.md), and ensure your SPL-token info uses the correct [Uniswap format](https://github.com/SolDev-stack/token-lists/blob/main/README.md).

File: src/tokens/solana.tokenlist.json [hunk 1]
```diff
@@ -68,7 +68,7 @@
         "social-token",
```
Comment (ashwinshetttty) [inline src/tokens/solana.tokenlist.json:68]: "ZIYOB",

**Review by ashwinshetttty — COMMENTED** [review]
Commit

**Review by ashwinshetttty — COMMENTED** [review]
Done.

**Pull request closed without merging**